lib/ES/Util.pm

```diff
@@ -260,7 +260,7 @@ sub _add_extra_title_page {
 #===================================
     my ( $index, $html_file ) = @_;
     my $extra_title_page = $index->basename;
-    $extra_title_page =~ s/\.a(scii)?doc$/-extra-title-page.html/ || die;
+    $extra_title_page =~ s/(\.x)?\.a(scii)?doc$/-extra-title-page.html/ || die;
```

Comment:
Like any good perl developer, I just solve the problem with regular expressions.